xcube/core/dsio.py

```diff
@@ -549,3 +553,28 @@ def rimraf(path):
         except OSError:
             warnings.warn(f"failed to remove file {path}")
             pass
+
+
+def _get_path_or_store(path, client_kwargs, mode, root):
+    path_or_store = path
+    anon_mode = True
+
+    if client_kwargs is not None:
+        if 'aws_access_key_id' in client_kwargs and 'aws_secret_access_key' in client_kwargs:
```

Comment:
I think it would be great if you could attempt to get the credentials from system env variables when the user does not
provide them in the client_kwargs. That is safer for docker containers and the like.

e.g.:

```
# Get the value of 'HOME' 
# environment variable 
key = 'HOME'
value = os.getenv(key) 
```

Reply:
I don't really agree, at least in this very case here, because users might want to write to different object storage providers with different credentials. For the CLI, this might make sense, Python API users can env vars on their own.

Reply:
I do not fully understand. Is this issue solved by the information, that this is just the key the s3fs library uses and that by defining the endpoint url it may also apply to OTC buckets? Or should I change something within the implementation? If so, I do need hints.

Reply:
Helge's suggestion is to use environment variables. But that prevents using different object storages at the same time, because environment variables are global.